verl/trainer/config/sft_trainer.yaml

```diff
@@ -32,7 +33,7 @@ model:
     cpu_offload: False
     offload_params: False
   external_lib: null
-  enable_gradient_checkpointing: False
+  enable_gradient_checkpointing: True
```

Comment:
Is this a intended breaking change?

Reply:
It won't affect training but will lower memory usage, allowing for a larger batch size or model size during SFT.